src/app/services/game-play.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, fromEvent } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { AnswerService } from './answer.service';
+import { Answer } from '../models/answer';
+
+interface Game {
+  answer: Answer;
+  acceptedGuesses: string[];
+  status: Status;
+  currentGuessIdx: number;
+  currentGuess: string;
+};
+
+enum Status {
+  playing,
+  won,
+  lost
+};
+
+const emptyGame: Game = {
+  answer: <Answer>{},
+  acceptedGuesses: [],
+  status: Status.playing,
+  currentGuessIdx: 0,
+  currentGuess: ''
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GamePlayService {
+  private readonly game$ = new BehaviorSubject<Game>(emptyGame);
+  constructor(private answerService: AnswerService) { this.startNewGame() }
+
+  public readonly currentGuess$ = this.game$.pipe(
+    map(game => game.currentGuess)
+  )
+  public readonly acceptedGuesses$ = this.game$.pipe(
+    map(game => game.acceptedGuesses)
+  )
+  public readonly currentGuessIdx$ = this.game$.pipe(
+    map(game => game.currentGuessIdx)
+  )
+
+  protected get game() {
+    return this.game$.getValue();
+  }
+
+  public addLetter(letter: string) {
+    if (this.game.currentGuess.length === 5) return;
+    let updatedGuess = this.game.currentGuess + letter
+
+    this.game$.next({
+      ...this.game,
+      currentGuess: updatedGuess
+    });
+  }
+
+  public removeLetter() {
+    if (this.game.currentGuess.length === 0) return;
+    let { currentGuess } = this.game;
```

Comment:
Same comment here: always prefer `const` over `let` and `var`. Also a suggestion (strictly a suggestion, totally up to your style preference), you can just put the `currentGuess.substring(0, currentGuess.length - 1) on line 68 to reduce the number of lines of code, but it's nothing really important, just a style preference thing